### PR TITLE
Portal/internal endpoints /v1/internal -> /api-internal/v1

### PR DESCRIPTION
**Pull request by robinkwilson**
To ensure the internal endpoints are private and secure, organize the endpoint under /api-internal/v1 and NOT /api/v1.

### CODE REVIEW

File: lib/ret_web/router.ex [hunk 1]
```diff
@@ -127,9 +128,9 @@ defmodule RetWeb.Router do
       resources("/slack", Api.V1.SlackController, only: [:create])
     end
 
-    scope "/v1/internal", as: :api_v1 do
+    scope "/api-internal/v1", as: :api_v1 do
```
Comment (netpro2k) [inline lib/ret_web/router.ex]: Is there a reason we want these routes on both `/api/api-internal/v1` and `/api-internal/v1`?

File: lib/ret_web/router.ex [hunk 2]
```diff
@@ -127,9 +128,9 @@ defmodule RetWeb.Router do
...
       pipe_through([:portal_header_auth])
-      get("/presence", Api.V1.PresenceController, :show)
+      get("/presence", ApiInternal.PresenceController, :show)
```
Comment (netpro2k) [inline lib/ret_web/router.ex]: Looks like you are missing V1 here?
```suggestion
      get("/presence", ApiInternal.V1.PresenceController, :show)
```

File: lib/ret_web/router.ex [hunk 3]
```diff
@@ -197,6 +197,16 @@ defmodule RetWeb.Router do
     forward "/", Absinthe.Plug, json_codec: Jason, schema: RetWeb.Schema
   end
 
+  scope "/api-internal", RetWeb do
+    pipe_through(
+      [:portal_header_auth, :secure_headers, :parsed_body, :api] ++ if(Mix.env() == :prod, do: [:ssl_only], else: [])
+    )
+
+    scope "/v1", as: :api_v1 do
```
Comment (netpro2k) [inline lib/ret_web/router.ex:200]: I forget what `as: :api_v1` does here but we depending on what it does we maybe want something like `as: api_internal_v1`. My guess is its for generating utility functions.

**Review by netpro2k — CHANGES_REQUESTED** [review]
We probably don't want the second copy of the route under `/api` and we should see what `as: :api_v1` does and consider changing it. Otherwise lgtm.

**Review by brianpeiris — APPROVED** [review]
LGTM

**Pull request merged**